ts/screens/wallet/payment/TransactionSummaryScreen.tsx

```diff
@@ -79,290 +91,110 @@ export type TransactionSummaryScreenNavigationParams = Readonly<{
-        },
-        buttonIndex => {
-          if (buttonIndex === 0) {
-            this.props.backToEntrypointPayment();
-            this.props.resetPayment();
-            showToast(
-              I18n.t("wallet.ConfirmPayment.cancelPaymentSuccess"),
-              "success"
-            );
-          }
-        }
-      );
-    } else {
-      this.props.navigation.goBack();
+});
+
+const renderFooter = (
+  isLoading: boolean,
+  error: TransactionSummaryError,
+  continueWithPayment: () => void,
+  help: () => void
+) => {
+  if (O.isSome(error)) {
+    const errorOrUndefined = O.toUndefined(error);
+    const errorType = getV2ErrorMainType(errorOrUndefined as DetailV2Keys);
+    switch (errorType) {
+      case "TECHNICAL":
+      case "DATA":
+      case "UNCOVERED":
+        return (
+          <ButtonSolid
+            onPress={help}
+            fullWidth
+            accessibilityLabel={I18n.t("payment.details.info.buttons.help")}
+            label={I18n.t("payment.details.info.buttons.help")}
+          />
+        );
+      // There's a strange case where error is 'some' but
+      // its value is undefined (e.g. network error).
+      // In this case we fallback to the 'continue' CTA
+      // so that the user can eventually retry.
+      case undefined:
+        break;
+      default:
+        return <></>;
     }
-  };
-
-  private getSecondaryButtonProps = () => ({
-    bordered: pot.isNone(this.props.paymentId),
-    cancel: pot.isSome(this.props.paymentId),
-    onPress: this.handleBackPress,
-    title: I18n.t("global.buttons.cancel")
-  });
-
-  private renderFooterSingleButton() {
-    return (
-      <FooterWithButtons
-        type={"SingleButton"}
-        leftButton={this.getSecondaryButtonProps()}
-      />
-    );
   }
-
-  private handleContinueOnPress = (verifica: PaymentRequestsGetResponse) => {
-    const { maybeFavoriteWallet, hasPayableMethods } = this.props;
-    if (hasPayableMethods) {
-      this.props.startOrResumePayment(
-        verifica,
-        maybeFavoriteWallet,
-        hasPayableMethods
-      );
-      return;
-    }
-    alertNoPayablePaymentMethods(this.props.navigateToWalletAddPaymentMethod);
-  };
-
-  private renderFooterButtons() {
-    const { potVerifica } = this.props;
-    const basePrimaryButtonProps = {
-      primary: true,
-      title: I18n.t("wallet.continue")
-    };
-
-    const primaryButtonProps =
-      pot.isSome(potVerifica) &&
-      !(pot.isLoading(potVerifica) || pot.isError(potVerifica))
-        ? {
-            ...basePrimaryButtonProps,
-            disabled: false,
-            onPress: () => this.handleContinueOnPress(potVerifica.value)
-          }
-        : {
-            ...basePrimaryButtonProps,
-            disabled: true
-          };
-
+  if (isLoading) {
+    // FIXME: Add a ButtonSolid with a loading spinner as soon as it's available by DS (https://pagopa.atlassian.net/browse/IOBP-302)
```

Comment:
When we press the button there is no evidence about the transaction being loaded. 
We should add something to show to the user that the app is loading and it's not being stuck somewhere. 
Maybe a LoadingSpinnerOverlay should be enough for now. What do you think?
Then, when IOBP-32 is merged, we could remove it in favor of a loading spinner inside the button.

Reply:
cc @thisisjp

Reply:
With the commit a5c23f84b06ab34c9389b603a4f3632c9ff83172 has been upgraded the latest io-app-design-system library version adding the new `ButtonSolid` prop `loading` in order to show a spinner while is in loading state.

Reply:
That's exactly what I was going to write. The `loading` state is available since `1.13.0`.